### PR TITLE
Upgrade deps and enable Expeditor

### PR DESCRIPTION
**Pull request by thommay**

### CODE REVIEW

**Comment by thommay** [comment]
The second commit is almost entirely mechanical

File: Gemfile [hunk 1]
```diff
@@ -7,12 +7,8 @@ group :pedant do
   gem "oc-chef-pedant", :git => "https://github.com/chef/chef-server.git"
 end
 
-group :changelog do
-  gem "github_changelog_generator", :git => "https://github.com/chef/github-changelog-generator.git"
```
Comment (tas50) [inline Gemfile]: beautiful

File: lib/chef_zero/endpoints/organization_association_request_endpoint.rb [hunk 2]
```diff
@@ -13,7 +13,6 @@ def delete(request)
         end
         username = $1
         path = request.rest_path[0..-2] + [username]
-        data = FFI_Yajl::Parser.parse(get_data(request, path))
```
Comment (tas50) [inline lib/chef_zero/endpoints/organization_association_request_endpoint.rb]: that's nice to not have to do anymore

File: Gemfile [hunk 3]
```diff
@@ -7,12 +7,8 @@ group :pedant do
...
-end
-
 group :development, :test do
-  gem "chefstyle", "= 0.3.1"
+  gem "chefstyle", git: "https://github.com/chef/chefstyle.git", branch: "master"
```
Comment (lamont-granquist) [inline Gemfile]: this should probably just float on the released version

**Pull request merged**